packages/core/android/src/main/java/io/sentry/react/RNSentryModuleImpl.java

```diff
@@ -239,9 +239,6 @@ protected void getSentryAndroidOptions(
     if (rnOptions.hasKey("sessionTrackingIntervalMillis")) {
       options.setSessionTrackingIntervalMillis(rnOptions.getInt("sessionTrackingIntervalMillis"));
     }
-    if (rnOptions.hasKey("shutdownTimeout")) {
-      options.setShutdownTimeoutMillis(rnOptions.getInt("shutdownTimeout"));
-    }
```

Comment:
Let's define the `shutdownTimeout` option in the RN TS Options since it's still relevant for closing the native SDKs.

Reply:
fixed on https://github.com/getsentry/sentry-react-native/pull/4568/commits/816b506dc3c8377405470eea8300041624acb0a3